config.py

```diff
@@ -92,7 +99,7 @@ class Config(object):
     USAGE_BATCH_SIZE = int(os.getenv("USAGE_BATCH_SIZE") or 1000)
     COURSES_BATCH_SIZE = int(os.getenv("COURSES_BATCH_SIZE") or 1000)
     TOPICS_BATCH_SIZE = int(os.getenv("TOPICS_BATCH_SIZE") or 1000)
-    COURSEWORK_BATCH_SIZE = int(os.getenv("COURSEWORK_BATCH_SIZE") or 120)
+    COURSEWORK_BATCH_SIZE = int(os.getenv("COURSEWORK_BATCH_SIZE") or 1000)
```

Comment:
Is this a default we want to change? For me the problem with 1000 size batches on coursework is that the request times out. Is that not happening for you?

Reply:
This was working fine in both domains now with the recent performance tweaks.